.github/workflows/tests_and_cov.yml

```diff
@@ -21,7 +21,7 @@ jobs:
 
       - name: Install deps
         run: |
-          pip install --upgrade --upgrade-strategy eager -r requirements-dev.txt -e .
+          pip install --upgrade --upgrade-strategy eager -r requirements-dev.txt -r requirements.txt -e .
```

Comment:
What changed here?

Reply:
Let's see, I think it didn't try to install the git+http from the requirements without giving it explicitly?

Reply:
Ah yes, right